scripts/accountQueries.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  getAccountTypes,
+  Governance,
+  Proposal,
+  ProposalState,
+} from '@models/accounts'
+import { GOVERNANCE_SCHEMA } from '@models/serialisation'
+import { PublicKey } from '@solana/web3.js'
+import { getConnectionContext } from '@utils/connection'
+import { AccountsQuery } from 'glasseater'
+
+const {
+  current: { commitment },
+  endpoint,
+} = getConnectionContext('mainnet')
+
+const connection: Parameters<AccountsQuery['fetch']> = [
+  { commitment, endpoint },
+  // xxx: if using public or throttled RPC, change/remove config below
+  {
+    msDelayBetweenBatchedRequests: 0,
+    maxNumberOfReqeustsPerBatch: 150,
+  },
+]
+
+/**
+ * Fetches the number of open Proposals in for a list of Realms
+ * @example
+ * // returns { "DPi...xFE": 1, "foo...123": 4 }
+ * `await getProposalCounts(mainnetRealms)`
+ * @returns {Object} Returns dictionary of `{ [realmId]: count }`
+ */
+export const getNumberOfProposalsInVotingState = async (realms) => {
+  // 1/3) get all governance accounts for each realm
+  const governancesQuery = new AccountsQuery(GOVERNANCE_SCHEMA, Governance)
```

Comment:
Note: the schema should be a function which resolves based on the account type to support different version of the program. See https://github.com/solana-labs/oyster/blob/6b48d97056b789129deec46591481f0074ff823c/packages/governance/src/models/core/api.ts#L75